### ISSUE TITLE
Wrong icons in a request

### ISSUE THREAD

**Issue by Primokorn** [danimahardhika/candybar-library#120]
Hi!

Here is the icon request I have received from a user:
> Manufacturer : HUAWEI
> Model : HUAWEI CAM-L21
> Product : CAM-L21
> Screen Resolution : 720 x 1208 pixels
> Android Version : 6.0
> App Version : 12.0
> CandyBar Version : 3.5.0-b4
> 
> 
> BSPlayer
> com.bsplayer.bspandroid.full/com.bsplayer.bsplayeran.BSPMain_new
> https://play.google.com/store/apps/details?id=com.bsplayer.bspandroid.full
> 
> Cikukua
> com.setio.budi.cikukua.icons/com.setio.budi.cikukua.icons.activities.SplashActivity
> https://play.google.com/store/apps/details?id=com.setio.budi.cikukua.icons
> 
> Hell Icon Pack
> com.setio.budi.darko.icons/com.setio.budi.darko.icons.activities.SplashActivity
> https://play.google.com/store/apps/details?id=com.setio.budi.darko.icons
> 
> MediaConverter
> com.AndroidA.MediaConverter/com.AndroidA.MediaConverter.MainActivity
> https://play.google.com/store/apps/details?id=com.AndroidA.MediaConverter
> 
> Sagon
> com.bandot.sagon.dark.iconpack/com.bandot.sagon.dark.iconpack.activities.SplashActivity
> https://play.google.com/store/apps/details?id=com.bandot.sagon.dark.iconpack

In the attachment the icons of the above apps don't match. They are icons of other apps that I support in my icon pack.

[icon_request_03_12_2017_17_29_46.zip](https://github.com/danimahardhika/candybar-library/files/1550752/icon_request_03_12_2017_17_29_46.zip)
 
It's the first time that I have this issue.
*Cikukua* has been requested by other users for another icon pack and the icon in the attachment is good.

How could this be possible?
Thank you.

**Comment by danimahardhika** [comment]
Never faced this issue before. I'm not sure what happened actually. I'll take a look into this issue.

**Comment by Primokorn** [comment]
Yes I have already received hundreds of icon requests but it's the first time that I notice it.
Let me know if you need more information. I could contact the user.
